py-polars/tests/unit/operations/arithmetic/test_arithmetic.py

```diff
@@ -688,16 +688,6 @@ def test_list_arithmetic_nulls(a: list[Any], b: list[Any], expected: list[Any])
 
 
 def test_list_arithmetic_error_cases() -> None:
-    # Different series length:
```

Comment:
I still think some of these tests are necessary? E.g. `pl.Series([[1, 2], [3, 4]]) + pl.Series([[1, 1], [2, 2], [3, 4]])` should complain.

(Separately I am honestly not super-excited about semantics of single-item Series working this way, but I guess that's how literals work? So if that design decision has already been made, oh well.)

Reply:
Or rather, some variations of existing tests should still be there, by tweaking existing assertions instead of deleting them.